packages/admin/admin-date-time/src/helpers/timePickerHelpers.ts

```diff
@@ -0,0 +1,71 @@
+const today = new Date();
+
+export const getDateRangeListByMinuteStep = (startTimeValue: string, endTimeValue: string, minuteStep: number): Date[] => {
```

Comment:
You could use [eachMinuteOfInterval](https://date-fns.org/v2.28.0/docs/eachMinuteOfInterval) for this helper